test/test_appscale_terminate_instances.py

```diff
@@ -303,6 +303,119 @@ def test_terminate_in_cloud_and_succeeds(self):
     AppScaleTools.terminate_instances(options)
 
 
+  def test_terminate_in_cloud_with_unreachable_node(self):
+    # let's say that there is a locations.json file with key
```

Comment:
This file was not meant to be committed... I'm going to remove the changes from this PR.